app/helpers/posts_helper.rb

```diff
@@ -1,15 +1,18 @@
 module PostsHelper
   def hero_image_tag_for(post)
     content_tag :div, class: 'post-hero' do
-      html = ''
-      if post.assets.any? && post.assets.cover.any?
-        html << image_tag(post.assets.cover.first.asset.url)
-      end
-      html << content_tag(:div, nil, class: 'hero-chevron')
-      raw(html)
+      raw(hero_image_html_for(post))
     end
   end
 
+  def hero_image_html_for(post)
+    html = ''
+    if post.assets.any? && post.assets.cover.any?
+      html << image_tag(post.assets.cover.first.asset.url)
+    end
+    html << content_tag(:div, nil, class: 'hero-chevron')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.